GitUI/Interops/NativeMethod.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Runtime.InteropServices;
```

Comment:
This must reside in https://github.com/gitextensions/gitextensions/tree/master/GitUI/Interops/User32 and follow the file and class naming schemes